src/database/schema/ChannelListMemberSchema.ts

```diff
@@ -102,6 +102,10 @@ class ChannelListMemberSchema implements BaseDbSchema {
             B.bio,
             B.is_banned,
             B.last_active_at,
+            B.anon_user_info_color_code,
+            B.anon_user_info_color_name,
+            B.anon_user_info_emoji_code,
+            B.anon_user_info_emoji_name,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `anon_user_info_color_code`, `anon_user_info_color_name`, `anon_user_info_emoji_code`, and `anon_user_info_emoji_name` have been added to the database schema. Please ensure that these fields are properly initialized and sanitized before use to prevent any potential SQL injection attacks or data inconsistencies. Also, make sure that these fields are handled correctly in all places where this schema is used, including insertions, updates, and reads.

```diff
+            B.anon_user_info_color_code,
+            B.anon_user_info_color_name,
+            B.anon_user_info_emoji_code,
+            B.anon_user_info_emoji_name,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->